### DISCUSSION TITLE
Surface form potential when using double layer capacitance

### DISCUSSION THREAD

**Discussion by Yanisiqi** [pybamm-team/PyBaMM#4538] [Q&A] [answered]
Hello everyone:
When I want to include double layer capacitance in DFN model, pybamm guildeline tells me to set surface potential option to be differential: model = pybamm.lithium_ion.DFN(options={'surface form': 'differential'}).
Then I notice that electrolyte current density i_e is calculated in the following way (full_surface_form_conductivity.py):
i_e = conductivity * (
                param.chiRT_over_Fc(c_e, T) * pybamm.grad(c_e)
                + pybamm.grad(delta_phi)
                + i_boundary_cc / sigma_eff
            )
 However, in normal DFN model, i_e is calculated in this way:
 i_e = (param.kappa_e(c_e, T) * tor) * (
            param.chiRT_over_Fc(c_e, T) * pybamm.grad(c_e) - pybamm.grad(phi_e)
        )
 These two equations are quite different. I wonder why double-layer capacitance modelling need to use a different i_e expression compared with normal DFN model? Can someone give some explanations or relevant papers? Thank you.
 
 Best regards

**Comment by valentinsulzer** [comment, marked as answer]
We have to rearrange in order to get a single equation in terms of `delta_phi` (which is `phi_s - phi_e`). We can do this by combining the equations for `i_e` and `i_s` using the fact that `i_e + i_s = I`. See equations 47 and 48a in https://iopscience.iop.org/article/10.1149/2.0301910jes/pdf

**Reply by Yanisiqi** [reply]
Thanks
